Failing tests

I've cloned the repository and tried running the tests, but I get the following output from pytest:

```
================================= short test summary info =================================
FAILED tests/test_api.py::test_weather_get - assert 404 == 200
FAILED tests/test_api.py::test_weather_list_get - assert 404 == 200
=============================== 2 failed, 3 passed in 6.02s ===============================
```

Since no instructions were given on how to properly run the tests, I've installed the `requirements.txt` dependencies using pip into a python 3.8 virtual environment and simply ran `pytest`, without parameters.

--- a/Pipfile
+++ b/Pipfile
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 flask = "~=1.1"
 flask-restful = "~=0.3"
-flask-caching = "~=1.9"
+flask-caching = "~=1.10"
 requests = "~=2.25"
 
 [dev-packages]

--- a/README.md
+++ b/README.md
@@ -71,6 +71,7 @@ It's the main interface of the application. It's responsible for providing the W
   This endpoint has no input parameters.
 
 - **Responses**
+
   This is the result for `http://localhost:5000/weather/Florianópolis`:
 
   ```json
@@ -91,6 +92,7 @@ It's the main interface of the application. It's responsible for providing the W
   |`max`|Define the number of cities to be request. This parameter must be a positive integer between 1 and 5. Ex.: `4`. If not informed, the default value will be `5`|integer|**no**|
 
 - **Responses**
+
   This is the result for `http://localhost:5000/weather` with `max = 4`:
 
   ```json
@@ -117,3 +119,70 @@ It's the main interface of the application. It's responsible for providing the W
     }
   ]
   ```
+
+## For Developers
+
+### Settings environment
+
+To configure your development environment, it's necessary that you have **Pipenv** installed on your machine.
+
+1. First, clone the remote application repository on your local machine:
+
+ ```bash
+ $ git clone https://github.com/rarygc/weather-buddy-api.git
+ ```
+
+2. Them, access the application's base directory:
+
+ ```bash
+ $ cd weather-buddy-api
+ ```
+
+3. Now, create a `.env` environment configuration file in the base directory with the following content:
+
+ ```env
+ # Flask App settings
+ FLASK_APP=api
+ FLASK_ENV=development
+ DEBUG=True
+
+ # OpenWeatherMap API settings
+ OPENWEATHERMAP_TOKEN=3f62164f8ce1f8cb5ae8e2d02918babb
+ ```
+ > **Note:** In the `.env` file there are configuration variables for the flask application and authentication in the OpenWeatherMap API. The `OPENWEATHERMAP_TOKEN` variable is receiving a default value that is an access token generated exclusively for this public application. But, feel free to generate your own access token at the [OpenWeatherMap Official Website](https://openweathermap.org) and use it, if desired.
+
+### Run the application
+
+Assuming that the previous step was properly performed, to run the application in development mode, just follow the guidelines below:
+
+1. First, install required packages with Pipenv:
+
+ ```bash
+ $ pipenv install
+ ```
+
+2. Finally, run the application with Flask:
+
+ ```bash
+ $ pipenv run flask run
+ ```
+
+ > **Note:** All commands related to a python environment, such as `pip freeze` or `python [file]`, are available, using the `pipenv run` command as a prefix to run any command within the environment.
+
+### Test the application with Pytest
+
+Assuming that the "Settings environment" step was properly executed, to run the application tests in development mode, just follow the guidelines below:
+
+1. First, install development required packages:
+
+ ```bash
+ $ pipenv install --dev
+ ```
+
+ > **Note:** This command installs all packages installed in the previous example, also including development packages listed in `[dev-packages]` in the `Pipfile`.
+
+2. Finally, run the application tests with Pytest:
+
+ ```bash
+ $ pipenv run pytest
+ ```

--- a/api/__init__.py
+++ b/api/__init__.py
@@ -6,7 +6,7 @@ from flask_caching import Cache
 app = Flask(__name__)
 
 # Create and configure Flask Cache instance
-cache = Cache(app, config={'CACHE_TYPE': 'simple', 'CACHE_DEFAULT_TIMEOUT': 300})
+cache = Cache(app, config={'CACHE_TYPE': 'flask_caching.backends.SimpleCache', 'CACHE_DEFAULT_TIMEOUT': 300})
 cache.init_app(app)
 
 # Create a Flask RESTful Api instance

